src/WebJobs.Script/Description/DotNet/FunctionAssemblyLoadContext.cs

```diff
@@ -234,6 +269,12 @@ protected override Assembly Load(AssemblyName assemblyName)
             return LoadCore(assemblyName);
         }
 
+        internal bool IsHigherThan(AssemblyName name1, AssemblyName name2)
```

Comment:
Why a custom method here when we should be able to rely on the version comparison? Are you explicitly trying to ignore the other elements?

Reply:
No... just a brain fart. I think I copied it from our custom logic we have for the policy and went with it.